tests/chainerx_tests/unit_tests/routines_tests/test_logic.py

```diff
@@ -225,12 +225,14 @@ def logical_and(xp, a, b):
 def logical_or(xp, a, b):
     return xp.logical_or(a, b)
 
+def logical_xor(xp, a, b):
+    return xp.logical_or(a, b)
```

Comment:
Hm, you're testing the wrong routine here.
```suggestion
    return xp.logical_xor(a, b)
```

Reply:
oops.